src/scripts/upload.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${AVIATOR_API_TOKEN}" ]; then
+    echo "AVIATOR_API_TOKEN is required."
+    exit 1
+fi
+
+if [ -z "${FILE_PATH}" ]; then
```

Comment:
you should also validate that the file even exists before calling this. Ideally if the file doesn't exist, we should just log a message and do `exit 0`